src/main/java/fr/esgi/al5_2/Tayarim/repositories/ProprietaireRepository.java

```diff
@@ -8,8 +8,28 @@
 
 import java.util.Optional;
 
+/**
+ * Interface de repository pour les opérations de base de données sur les entités Proprietaire.
+ * Fournit des méthodes pour retrouver un propriétaire par email ou par numéro de téléphone.
+ */
 @Repository
-public interface ProprietaireRepository extends JpaRepository<Proprietaire, Long>{
-    Optional<Proprietaire> findFirstByEmail(@NonNull String email);
-    Optional<Proprietaire> findFirstByNumTel(@NonNull String numTel);
+public interface ProprietaireRepository extends JpaRepository<Proprietaire, Long> {
+
+  /**
+   * Recherche le premier propriétaire correspondant à l'adresse email spécifiée.
+   *
+   * @param email L'adresse email utilisée pour la recherche.
+   * @return Un Optional contenant le propriétaire trouvé ou un Optional vide si aucun propriétaire
+   * n'est trouvé.
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocTagContinuationIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line continuation have incorrect indentation level, expected level should be 4.